### PR TITLE
Use new query parameters to benefit from the shorter response

### PR DESCRIPTION
**Pull request by chrode**

### CODE REVIEW

File: yo/app/scripts/controllers/data-list.js [hunk 1]
```diff
@@ -123,15 +123,15 @@ angular.module('dmpApp')
                 $scope.files = '';
             });
 
-            DataModelResource.query(function(results) {
+            DataModelResource.query({ format : 'medium' }, function(results) {
 
-                $scope.models = loDash.filter(results, 'data_resource');
+                $scope.models = loDash.filter(results, 'dataResource');
```
Comment (knutwalker) [inline yo/app/scripts/controllers/data-list.js:128]: The case change for data_resource here was not intentional, should I revert it?

**Comment by knutwalker** [comment]
I think there is something missing. When creating a new project, you shoul load the full data model since you need the schema, APs, etc.

File: yo/app/scripts/controllers/data-list.js (hunk 1 above)
Comment (chrode) [inline yo/app/scripts/controllers/data-list.js:128]: Yes.

**Comment by chrode** [comment]
Totally missed that. Thx. Will fix this.

**Pull request closed without merging**